app/core/vueRouter.ozar.js

```diff
@@ -110,6 +110,11 @@ export default function getVueRouter () {
           path: '/efficacy',
           component: () => import(/* webpackChunkName: "efficacy" */ 'app/views/efficacy/EfficacyView.vue')
         },
+        { // has no use in Oz but keeping to have divergence between files less
```

Comment:
The comment itself is a divergence that we will have to merge later! :p
```suggestion
        {
```